.github/workflows/wfc_lifecycle.yml

```diff
@@ -92,26 +92,30 @@ jobs:
     steps:
       - name: mark issues as stale
         run: |
-          before=`date -d '${{ inputs.daysBeforeStale }} days ago' --iso-8601`
-          echo "Getting all issues untouched since: $before"
-          issues=`gh issue list -S "-label:${{ inputs.staleLabel }} updated:<$before" -s open -R ${{ github.repository }} --json number -t '{{range .}}{{.number}} {{end}}'`
-          echo "Updating `echo $issues | wc -w` issues"
-          for issue in $issues; do
-            gh issue edit $issue --add-label ${{ inputs.staleLabel }} -R ${{ github.repository }}
-            gh issue comment $issue -R ${{ github.repository }} --body "$COMMENT"
-          done
+          if [[ `gh issue list --limit 1 --repo ${{ github.repository }} 2> /dev/null` ]]; then
```

Comment:
I _think_ you should be able to just write `if gh issue ...` to test the exit code.

Reply:
You’re right, I simplified to the following:

```diff
- if [[ `gh issue list --limit 1 --repo ${{ github.repository }} 2> /dev/null` ]]; then
+ if gh issue list --limit 1 --repo ${{ github.repository }} > /dev/null 2>&1; then
```